src/components/Nav.js

```diff
@@ -7,7 +7,9 @@ function Nav() {
       <h1 className="theHeading">Bookstore CMS</h1>
       <nav className="options">
         <Link to="/">BOOKS</Link>
-        <Link to="Categories">CATEGORIES</Link>
+        <Link className="cate" to="Categories">
+          CATEGORIES
+        </Link>
```

Comment:
[OPtIONAL]
- Your navbar is missing the profile avatar. KIndly add it 

![image](https://user-images.githubusercontent.com/63473584/224518090-5a79df16-19f9-49b0-a17b-fcf3a527de73.png)